src/vdaf/poplar1.rs

```diff
@@ -2391,25 +2406,37 @@ mod tests {
     #[ignore]
     #[test]
     fn test_vec_poplar1_0() {
-        check_test_vec(include_str!("test_vec/08/Poplar1_0.json"));
+        check_test_vec(include_str!("test_vec/13/Poplar1_0.json"));
```

Comment:
FWIW we ended up dropping the version number from the path in the spec repo: https://github.com/cfrg/draft-irtf-cfrg-vdaf/tree/main/test_vec/vdaf

It might help make updating easier if we mirror the directory structure there. Though TBH I think it's fine as is.